app/views/notifications/started.text.erb

```diff
@@ -0,0 +1,14 @@
+<%= @owned_by.name %> has started your story '<%= @story.title %>'.
+
+<% if (@story.estimate.nil? || @story.estimate == 0) %>
```

Comment:
@talyssonoc [0 is not false in Ruby](http://stackoverflow.com/q/10387515/1042324)